sklearn/linear_model/tests/test_coordinate_descent.py

```diff
@@ -247,6 +268,132 @@ def test_lasso_cv_positive_constraint():
     assert min(clf_constrained.coef_) >= 0
 
 
+@pytest.mark.parametrize(
+    "LinearModel, params",
+    [(Lasso, {"tol": 1e-16, "alpha": 0.1}),
+     (LassoLars, {"alpha": 0.1}),
+     (RidgeClassifier, {"solver": 'sparse_cg', "alpha": 0.1}),
+     (ElasticNet, {"tol": 1e-16, 'l1_ratio': 1, "alpha": 0.1}),
+     (ElasticNet, {"tol": 1e-16, 'l1_ratio': 0, "alpha": 0.1}),
+     (Ridge, {"solver": 'sparse_cg', 'tol': 1e-12, "alpha": 0.1}),
+     (BayesianRidge, {}),
+     (ARDRegression, {}),
+     (OrthogonalMatchingPursuit, {}),
+     (MultiTaskElasticNet, {"tol": 1e-16, 'l1_ratio': 1, "alpha": 0.1}),
+     (MultiTaskElasticNet, {"tol": 1e-16, 'l1_ratio': 0, "alpha": 0.1}),
+     (MultiTaskLasso, {"tol": 1e-16, "alpha": 0.1}),
+     (Lars, {}),
+     (LinearRegression, {}),
+     (LassoLarsIC, {})]
+ )
+def test_model_pipeline_same_as_normalize_true(LinearModel, params):
+    # Test that linear models (LinearModel) set with normalize set to True are
+    # doing the same as the same linear model preceeded by StandardScaler
+    # in the pipeline and with normalize set to False
+
+    # normalize is True
+    clf_norm = LinearModel(normalize=True, fit_intercept=True, **params)
+
+    clf_pipe = make_pipeline(
+        StandardScaler(),
+        LinearModel(normalize=False, fit_intercept=True, **params)
+    )
+
+    is_multitask = clf_norm._get_tags().get("multioutput_only", False)
+
+    # prepare the data
+    n_samples, n_features = 100, 2
+    rng = np.random.RandomState(0)
+    w = rng.randn(n_features)
+    X = rng.randn(n_samples, n_features)
+    X += 20  # make features non-zero mean
+    y = X.dot(w)
+
+    # make classes out of regression
+    if is_classifier(LinearModel):
```

Comment:
```suggestion
    if is_classifier(model):
```
`is_classifier` expects an instance instead of a class. Otherwise we need to use `issubclass(LinearModel, ClassifierMixin)`